procurement/templates/procurement/includes/tender_filter.html

```diff
@@ -1,14 +1,14 @@
 {% load bootstrap4 %}
 
-            <fieldset class="col-sm-auto">
+            <fieldset class="d-flex align-items-end flex-wrap mb-md-2">
                 {% bootstrap_field filter.form.sort %}
```

Comment:
This `bootstrap_field` tag is currently outputting a `<select class="form-control">` when it should be a `<select class="form-select">` … how can we fix that?

Reply:
@alexander-griffen Would it be helpful to have a chat about this today?